tests/custom_maven_install/regression_testing_install.json

```diff
@@ -1390,16 +1426,22 @@
       },
       "version": "1.5.0"
     },
+    "org.mockito:mockito-core": {
+      "shasums": {
+        "jar": "4be648c50456fba4686ba825000d628c1d805a3b92272ba9ad5b697dfa43036b"
+      },
+      "version": "3.3.3"
+    },
     "org.objenesis:objenesis": {
       "shasums": {
-        "jar": "c74330cc6b806c804fd37e74487b4fe5d7c2750c5e15fbc6efa13bdee1bdef80"
+        "jar": "5e168368fbc250af3c79aa5fef0c3467a2d64e5a7bd74005f25d8399aeb0708d"
       },
-      "version": "2.1"
+      "version": "2.6"
     },
     "org.openjfx:javafx-base": {
       "shasums": {
         "jar": "c5084a74417a89c69a0c122fae96a4b70bf619fc3d6218ea102a4047ec85ad04",
-        "linux": "2ebf6fa2cbbe1c8b4f7780e06e97beb038f644d5ecf9f15a41c5e88ee0ef9cf1"
+        "mac": "2d8052a08fd2e5d98e1d5a16d724ea5dd02102879de20a193225f57199803983"
```

Comment:
This is unfortunate, but I don't have a Linux machine/image with all required Bazel deps to regenerate this for Linux and have matching hashes